draft-ietf-rats-reference-interaction-models.md

```diff
@@ -300,7 +304,7 @@ The following subsections introduce and illustrate the interaction models:
 3. Streaming Remote Attestation
 
 Each section starts with a sequence diagram illustrating the interactions between Attester and Verifier.
-While the presented interaction models focus on the conveyance of Evidence, the intention of this document is in support of future work that applies the presented models to the conveyance of other Conceptual Messages, namely Attestation Results, Endorsements, Reference Values, or Appraisal Policies.
+While the presented interaction models focus on the conveyance of Evidence, the intention of this document is in support of any type of Conceptual Messages conveyance that is based on the presented models.
```

Comment:
Unclear how the document can _"[s]upport of any type of Conceptual Messages conveyance that is based on the presented models."_ without describing in practice how the models can be applied to other kinds of flows/CMs.

Reply:
see comment below, failed to reply here

Reply:
https://github.com/ietf-rats-wg/draft-ietf-rats-reference-interaction-models/pull/56/files#r1799833686